sycl/source/detail/helpers.cpp

```diff
@@ -6,8 +6,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <CL/sycl/detail/context_impl.hpp>
 #include <CL/sycl/detail/helpers.hpp>
+
+#include <CL/sycl/detail/context_impl.hpp>
 #include <CL/sycl/event.hpp>
 
```

Comment:
Please, keep the order sorted.

Reply:
They are ordered. I've just highlighted "main module header" - according to [LLVM coding guidelines](https://llvm.org/docs/CodingStandards.html#include-style):

> The Main Module Header file applies to `.cpp` files which implement an interface defined by a `.h` file. This `#include` should always be included **first** regardless of where it lives on the file system.

I can merge it back and re-order if you want

Reply:
I missed that it was done in `helpers.cpp`. 
Please, disregard this comment.